xclim/indicators/atmos/_temperature.py

```diff
@@ -781,11 +782,36 @@ def cfcheck(tas, snd=None):
     description="Temperature-based index used to estimate the development of corn crops. "
     "Corn growth occurs when the minimum and maximum daily temperature both exceeds "
     "specific thresholds : Tmin > {thresh_tasmin} and Tmax > {thresh_tasmax}.",
+    var_name="chu",
```

Comment:
haha, hého. You can't hide almost-breaking unrelated changes like this! ;)

I agree with it, let's just notify @marielabonte that the output of `corn_heat_units` will change with this PR.

Reply:
Woops, fair enough. Breaking changes mentioned above!